src/isar/config/settings.py

```diff
@@ -89,11 +89,10 @@ class Settings(BaseSettings):
     # Client ID for the OpenAPI client
     OPENAPI_CLIENT_ID: str = Field(default="5f412c20-8c36-4c69-898f-d2b5051f5fb6")
 
-    # MQTT username
+    # MQTT credentials
     # The username and password is set by the MQTT broker and must be known in advance
```

Comment:
Isn't the username and password *used by* the MQTT broker? And *set by* the user?

Reply:
Absolutely, correcting to _used_

Reply:
I think the reason for the wording is that the MQTT broker is another "agent" than this ISAR package, and it is in the MQTT broker that the user actually sets the password. Still, we avoid the confusion by writing _used_ here. Well observed.

Reply:
```python
    # The username and password for connecting to the MQTT broker
```